01-messaging-crypto-and-init.md

```diff
@@ -20,24 +20,21 @@ the unicode code point for LIGHTNING.<sup>[2](#reference-2)</sup>
 
 All messages are of form:
 
-1. `4-byte` big-endian data length.
-2. `4-byte` big-endian type.
-3. Data bytes as specified by the length.
+1. `2-byte` big-endian type.
+2. 4 bytes of padding.
+3. Data bytes as specified by the total packet length.
 
 All data fields are big-endian unless otherwise specified.
 
 ### Requirements
 
-A node MUST NOT send a message with data length greater than `8388608`
+A node MUST NOT send a message with more than `65529` data
```

Comment:
Yep, that looks like a definition slip.

We should probably spell out terms and use them consistently, eg:
1. Header: Encrypted 2 byte length (+2 byte type if we decide to go that way)
2. Header MAC: 16 byte mac of header.
3. Ciphertext: encrypted message as defined in other BOLTs.
4. Cipertext MAC: 16 byte mac of ciphertext.

(Better terms welcome!)